src/libstd/Cargo.toml

```diff
@@ -13,6 +13,7 @@ crate-type = ["dylib", "rlib"]
 alloc = { path = "../liballoc" }
 alloc_jemalloc = { path = "../liballoc_jemalloc", optional = true }
 alloc_system = { path = "../liballoc_system" }
+alloc_frame = { path = "../liballoc_frame" }
```

Comment:
@brson: Redox is using the `libc` allocator right now, so this would not be an issue.